### PR TITLE
Adding "python-frictionqpotspringblock" and "python-frictionqpotfem"

### PR DESCRIPTION
**Pull request by tdegeus**
Python API of "frictionqpotspringblock" and "frictionqpotfem"

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by tdegeus** [comment]
Ready for review @conda-forge/help-python-c

File: recipes/python-frictionqpotfem/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - pyxtensor
+    - goosefem
+    - gmattensor
+    - gmatelastoplasticqpot
+  run:
+    - python
+
+test:
+  imports:
+    - {{ name }}
```
Comment (chrisburr) [inline recipes/python-frictionqpotfem/meta.yaml:34]:
```suggestion
    - {{ name }}
    commands:
      - pip check
    requires:
      - pip
```

File: recipes/python-frictionqpotspringblock/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - pip
+    - pyxtensor
+    - qpot
+    - goosefem
+  run:
+    - python
+
+test:
+  imports:
+    - {{ name }}
```
Comment (chrisburr) [inline recipes/python-frictionqpotspringblock/meta.yaml:33]:
```suggestion
    - {{ name }}
    commands:
      - pip check
    requires:
      - pip
```

**Comment by tdegeus** [comment]
@chrisburr Your suggestion introduces a check on compile-time dependencies that are not needed on run-time. I therefore removed them.

**Comment by chrisburr** [comment]
If it's in `install_requires` then `pip` will always have issues as a result of these packages being incorrectly packaged: https://github.com/tdegeus/FrictionQPotFEM/blob/af47d14b3d4511d32cafd8e81f185ba1a4128e05/setup.py#L54

If you want to avoid putting the dependencies in `run` you need to make a new release that uses `setup_requires` for these dependencies.

**Comment by tdegeus** [comment]
I see @chrisburr , that clarifies things! Let me fix the repo

**Comment by tdegeus** [comment]
Ready for review @conda-forge/help-python-c

**Pull request merged**